core/src/test/java/com/datastax/dse/driver/internal/core/data/geometry/DefaultLineStringTest.java

```diff
@@ -94,13 +95,17 @@ public void should_load_from_well_known_binary() {
   }
 
   @Test
-  public void should_parse_valid_geo_json() {
-    assertThat(LineString.fromGeoJson(json)).isEqualTo(lineString);
+  public void should_create_and_parse_geo_json() {
+
+    String geoJsonStr = lineString.asGeoJson();
+    assertThat(JSONUtils.isValidJsonMap(geoJsonStr)).isTrue();
+    LineString newLineString = LineString.fromGeoJson(geoJsonStr);
+    assertThat(newLineString).isEqualTo(lineString);
```

Comment:
Not precisely what it says on the tin anymore.  We're validating the following:

1. We can generate JSON through a method called "asGeoJson"
2. The JSON so generated is valid
3. We can use this JSON to construct a new geometry type that matches up to the original

This doesn't strictly validate that we're generating GeoJSON but it _does_ validate that we're generating something that the ESRI datatypes can work with.  Seems like that's what we're really interested in here; we're then deferring the question of whether that JSON matches up to the GeoJSON spec to the ESRI impl.

I'm very open to the idea that renaming this method to reflect these changed expectations might be a good idea.